Davi Lima de Andrade - Estrutura de Dados Avançada

Davi-Andrade-30 opened the pull request.

Davi-Andrade-30 committed:
Add files via upload

Davi-Andrade-30 committed:
Update exercicio3_fila_reversao.py

Davi-Andrade-30 committed:
Update exercicio4_arvore_pre_ordem.py

Davi-Andrade-30 committed:
Update exercicio5_arvore_filhos.py

marcmec commented on exercicio2_fila_simulacao.py:
não recebe um array davi.

Davi-Andrade-30 replied:
Posso entregar até quando?

Davi-Andrade-30 replied:
O corrigido?

Davi-Andrade-30 replied:
Ou n tem mais jeito? 😔

Davi-Andrade-30 replied:
Refaço hj a noite ainda

marcmec reviewed (approved):
4/5

marcmec reviewed (changes requested):
refaça exercicio 2

Davi-Andrade-30 committed:
Update exercicio2_fila_simulacao.py

Atualização do exercicio 2, com o array agr, esqueci de colocar prof :(, foi mal ai qualquer coisa e bom final de semana, o resto dele pelo menos S2
